index.html

```diff
@@ -713,7 +803,7 @@ <h2>Generate a network error report</h2>
         policy</a>, and queues it for delivery.
       </p>
 
-      <ol>
+      <ol class="algorithm">
 
         <li>
           If the result of executing the <a>is-origin-trustworthy</a> algorithm

```

Comment:
"potentially trustworthy" would've been more appropriate

Reply:
This step requires the result to be "Potentially Trustworthy".  Do you mean simplifying the text to just:

> If request's origin is not potentially trustworthy

and not mentioning the name of the algorithm?

Reply:
Went ahead and did it here.